backend/fms_core/viewsets/_constants.py

```diff
@@ -142,5 +142,6 @@
     "id": PK_FILTERS,
     "name": CATEGORICAL_FILTERS_LOOSE,
     "index_set__name": CATEGORICAL_FILTERS_LOOSE,
+    "index_set__id": CATEGORICAL_FILTERS_LOOSE,
```

Comment:
PK_FILTERS ?

Reply:
I believe I changed this to CATEGORICAL_FILTERS_LOOSE no?